test_lib/scylla_bench_tools.py

```diff
@@ -17,7 +18,8 @@ def create_scylla_bench_table_query(compaction_strategy=None):
                     ck bigint,
                     v blob,
                     PRIMARY KEY (pk, ck)
-                ) WITH CLUSTERING ORDER BY (ck DESC)
+                ) WITH CLUSTERING ORDER BY (ck {})
+                    """.format(clustering_order) + """
```

Comment:
this `.format()` can enter in the end of the string... so you don't have to break it and use `""" + """`

Reply:
why not using f-string format?

Reply:
> why not using f-string format?

that would be another option for it, but for sure not what is in there...

Reply:
not forget to escape the "{}" that must be kept as string

Reply:
fixed